public/res/51-smartx.conf

```diff
@@ -0,0 +1,22 @@
+# Generic template for all ALSA <-> SmartXbar plugins
+# Can be used to dynamically create an ALSA to SmartXbar adapter by using it like this:
+# aplay -Dsmartx:MyNewDev ~/Music/<some wav file>
+
+pcm.smartx {
+  @args [ NAME ]
+  @args.NAME {
+    type string
+  }
+  type smartx
+  name $NAME
+}
+
+pcm_type.smartx
+{
+	lib "<lib_path>/libasound_module_pcm_smartx.so"
```

Comment:
Why not add here with the `@@` tag? So it's one less change on the next patch. And also add with the right name...

Reply:
oops. I am not supposed to push this file. let me check

Reply:
oh you were looking at the outdated file. it's changed into 51-smartx.conf.in file.